### ISSUE TITLE
"could not find process" error after successful convox run

### ISSUE THREAD

**Issue by thieman** [convox/rack#2613]
We're using AWS. We saw the following:

```
convox run ... # no flags beyond specifying rack and app
# then we saw the entire output of the run, which ended successfully
ERROR: could not find process
```

I haven't really started to dig into this yet, putting it here in case anybody else is seeing this.

**Comment by ddollar** [comment]
Please paste some rack logs here if you happen to see this again.

**Comment by ddollar** [comment]
This is fixed in #2379
